sw.js

```diff
@@ -10,16 +10,17 @@ const OFFLINE_FILES = [
     "img/screenshot.png",
     "img/mobile_screenshot.png",
     "https://cdn.emulatorjs.org/versions.json",
-
-    // Only cache stable assets offline...?
     "https://cdn.emulatorjs.org/stable/data/loader.js",
+    "https://cdn.emulatorjs.org/stable/data/version.json",
```

Comment:
I forgot to complete this, but the caching process will be consistent with the other files. I'll compare the cache version to the main version file to determine if the cache needs updating.

Reply:
So it'll pull on every load? The way it is currently set up causes duplicate data with EmulatorJS, since you dont have `window.EJS_disableDatabases` defined and EmulatorJS is built to cache the cores by default.